dataset.py

```diff
@@ -453,3 +454,75 @@ def create_fraudulent_samples(real_samples_data: pd.DataFrame, mean_isoscapes: l
     count += 1
 
   return fake_sample
+
+def valid_in_all_rasters(
```

Comment:
Please change this to _valid_in_all_rasters to reflect that this should be not used outside of this module (it's a helper method).